hc/front/tests/test_add_pushbover.py

```diff
@@ -38,3 +38,14 @@ def test_it_validates_nonce(self):
         assert r.status_code == 403
 
     ### Test that pushover validates priority
+
+    def test_it_validates_priority(self):
```

Comment:
Couldnt **test_pushover_validates_priority** be a more descriptive method name ?